aea/configurations/base.py

```diff
@@ -1421,9 +1441,12 @@ def json(self) -> Dict:
                 CONTRACTS: sorted(map(str, self.contracts)),
                 PROTOCOLS: sorted(map(str, self.protocols)),
                 SKILLS: sorted(map(str, self.skills)),
-                "default_connection": str(self.default_connection)
-                if self.default_connection is not None
-                else None,
+                CUSTOMS: sorted(map(str, self.customs)),
+                "default_connection": (
```

Comment:
best to make this a constant `default_connection`

Reply:
This is limited to the agent config only since it's a config class argument, if we do this for default_connection we'll have to do this for every argument

https://github.com/valory-xyz/open-aea/blob/abb8a8903f3639d4e5b49073527f86ea2b7b8643/aea/configurations/base.py#L1432-L1439

https://github.com/valory-xyz/open-aea/blob/abb8a8903f3639d4e5b49073527f86ea2b7b8643/aea/configurations/base.py#L1432-L1439